pkgs/development/python-modules/pygame-gui/default.nix

```diff
@@ -9,14 +9,14 @@
 
 buildPythonPackage rec {
   pname = "pygame-gui";
-  version = "0.6.4";
+  version = "064";
```

Comment:
Please bring back the dots. Upstream advertises them with dots, only the tags are weird https://github.com/MyreMylar/pygame_gui/releases/tag/v_064

Reply:
Will drop that commit 689a2ecafd64c77dfd1b45d3f606d26420f5a71c in the next rebase.